mindsworth/My-Diary-Frontend
# #162392096 User should be able to view dashboad

#### What does this PR do?
A logged in user can view the dashboard

#### Description of Task to be completed?
- Fix the responsiveness
- the user can view the dashboard when logged in

#### How should this be manually tested?
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 8080
3. visit the right URL e.g http://localhost:8080 to see the React Setup.
3. login and it redirect to the dashboard page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[ #162392096](https://www.pivotaltracker.com/story/show/162392096)

#### Screenshots (if appropriate)
<img width="1680" alt="screenshot 2018-12-04 at 12 24 20 pm" src="https://user-images.githubusercontent.com/14542449/49438771-92e38a80-f7bf-11e8-92c0-e91b5c3b87ff.png">

#### Questions:
N/A
